internal/exp/metrics/staleness/map.go

```diff
@@ -7,24 +7,11 @@ import (
 	"time"
```

Comment:
This can be deleted, since it's moved to streams. Correct? 

And HashMap is identical to RawMap